study.md

```diff
@@ -21,18 +21,20 @@ There is no requirement to follow any links from these pages.
 After doing all the required readings please tell us anything you were confused about.
 
 ```md
-<!-- your answer here -->
+A callback lets you put in a function to be called later so it doesn't run right away?
+Why would I want to do a callback? If code has to wait for input before running e.g. a 'click'?
 ```
 
 Are there any nagging questions relating to functions that you have?
 
 ```md
-<!-- your answer here -->
+I think that the reasons to use a function expression is for a callback or when assigned to a variable?
```

Comment:
You can use a function declaration or function expression to pass into another function as a callback if you think that it'll lead to cleaner code.  Here's an example.
```javascript
const array = ['a', 'b', 'c'];

// Anonymous function as a callback function.
array.forEach((element, index) => {
  console.log(`The element of the array at index ${index} is ${element}.`);
});
// Output:
// The element of the array at index 0 is a.
// The element of the array at index 1 is b.
// The element of the array at index 2 is c.

// Function declaration for showElementInfo().
function showElementInfo(element, index) {
  console.log(`The element of the array at index ${index} is ${element}.`);
}

// Named function as a callback function.
array.forEach(showElementInfo);
// Output:
// The element of the array at index 0 is a.
// The element of the array at index 1 is b.
// The element of the array at index 2 is c.
```